### ISSUE TITLE
Publish as NPM package?

### ISSUE THREAD

**Issue by dit7ya** [mouse484/monaco-svelte-kit#7]
Would it be possible to publish this as a package to NPM so that this can used in other projects as a simple component?

**Comment by mouse484** [comment]
I have a plan.

**Comment by mouse484** [comment]
But, This project is a prototype. I need more time.

**Comment by dit7ya** [comment]
Thanks! It will also be very useful if there were some code walkthrough or commentary on what each piece of file is doing.

**Comment by mouse484** [comment]
Yes, I plan to write an article about this in Japanese within a month.


The notes is [here](https://zenn.dev/mouse_484/scraps/c0371acf4edc08)(Japanese)

**Comment by dit7ya** [comment]
Thanks! Unfortunately I cannot read Japanese. Could you please kindly write a very brief summary of what are the major things you had to do to get Monaco working in SvelteKit?

**Comment by mouse484** [comment]
Of course.

**Comment by mouse484** [comment]
But I'm trying a new approach now (@monaco-editor/loader), so it may change from the current code.
This one looks easy.

https://github.com/mouse484/monaco-editor-utils/tree/main/examples
